test/gurka/gurka.cc

```diff
@@ -64,9 +64,9 @@ std::vector<midgard::PointLL> to_lls(const nodelayout& nodes,
  */
 std::string build_valhalla_request(const std::string& location_type,
                                    const std::vector<midgard::PointLL>& waypoints,
-                                   const std::string& costing = "auto",
-                                   const std::unordered_map<std::string, std::string>& options = {},
-                                   const std::string& stop_type = "break") {
+                                   const std::string& costing,
+                                   const std::unordered_map<std::string, std::string>& options,
+                                   const std::string& stop_type) {
```

Comment:
I made this function public so it can be used in tests. Hope that's fine with you..